pkg/db-repo/orm.go

```diff
@@ -19,14 +19,21 @@ type OrmSettings struct {
 	Application string              `cfg:"application" default:"{app_name}"`
 }
 
+type OrmConfig struct {
+	DbName string `cfg:"db" default:"default"`
+}
+
 func NewOrm(config cfg.Config, logger mon.Logger) (*gorm.DB, error) {
-	dbClient, err := db.NewClient(config, logger, "default")
+	dbName := OrmConfig{}
+	config.UnmarshalKey("orm", &dbName)
+
+	dbClient, err := db.NewClient(config, logger, dbName.DbName)
```

Comment:
What's the use case behind this?

Reply:
This is the use case behind this:
https://gitlab.applike-services.info/mcoins/backend/marketing/-/issues/1901